features/attach_validtoken.feature

```diff
@@ -398,9 +398,11 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Pro
        """
        And I append the following on uaclient config:
        """
+         update_contract_info_timer: 0
        features:
          machine_token_overlay: "/tmp/machine-token-overlay.json"
        """
+       And I run `python3 /usr/lib/ubuntu-advantage/timer.py` with sudo
```

Comment:
0 is treated as a special value to disable the timer job, so I wouldn't expect this to execute the new job :upside_down_face: 

To force a job to run in tests we normally `rm /var/lib/ubuntu-advantage/jobs-status.json` and then execute the timer scripts